geolib/models/dsheetpiling/loads.py

```diff
@@ -70,7 +70,7 @@ class UniformLoad(BaseDataClass):
     name: constr(min_length=1, max_length=50)
     left_load: float
     right_load: float
-    verification_load_settings: VerificationLoadSettings = VerificationLoadSettings()
+    verification_load_settings: VerificationLoadSettingsHorizontalLineLoad = VerificationLoadSettingsHorizontalLineLoad()
```

Comment:
Is it OK that the default value of load type changes from FAVOURABLE to DSHEETPILING_DETERMINED?

Reply:
I think having DSHEETPILING_DETERMINED as a default is more logical then having FAVOURABLE for the case with 3 options.